Missing files detected on override_dh_install

`pdebuild` has been not successfully finished due to missing installation files.

```
$ sudo pdebuild
...
[0/1] Installing files.
Installing nnstreamer-check to /build/nnstreamer-1.7.0.0/debian/tmp/usr/local/bin
make[1]: Leaving directory '/build/nnstreamer-1.7.0.0'
   debian/rules override_dh_install
make[1]: Entering directory '/build/nnstreamer-1.7.0.0'
dh_install --sourcedir=debian/tmp --list-missing
dh_install: nnstreamer-util missing files: /usr/bin/nnstreamer-check
dh_install: usr/include/nnstreamer/ml-api-common.h exists in debian/tmp but is not installed to anywhere
dh_install: usr/include/nnstreamer/tizen_error.h exists in debian/tmp but is not installed to anywhere
dh_install: usr/include/nnstreamer/nnstreamer.h exists in debian/tmp but is not installed to anywhere
dh_install: usr/include/nnstreamer/nnstreamer-single.h exists in debian/tmp but is not installed to anywhere
dh_install: usr/lib/x86_64-linux-gnu/libcapi-nnstreamer.so.1.7.0 exists in debian/tmp but is not installed to anywhere
dh_install: usr/lib/x86_64-linux-gnu/libcapi-nnstreamer.so.1 exists in debian/tmp but is not installed to anywhere
dh_install: usr/local/bin/nnstreamer-check exists in debian/tmp but is not installed to anywhere
dh_install: missing files, aborting
debian/rules:67: recipe for target 'override_dh_install' failed
make[1]: *** [override_dh_install] Error 2
make[1]: Leaving directory '/build/nnstreamer-1.7.0.0'
debian/rules:36: recipe for target 'binary' failed
make: *** [binary] Error 2
dpkg-buildpackage: error: fakeroot debian/rules binary gave error exit status 2
I: copying local configuration
E: Failed autobuilding of package
I: unmounting dev/pts filesystem
I: unmounting run/shm filesystem
I: unmounting proc filesystem
I: cleaning the build env 
I: removing directory /var/cache/pbuilder/build/10542 and its subdirectories
```

--- a/debian/nnstreamer-dev.install
+++ b/debian/nnstreamer-dev.install
@@ -8,5 +8,9 @@
 /usr/include/nnstreamer/tensor_typedef.h
 /usr/include/nnstreamer/tensor_filter_cpp.hh
 /usr/include/nnstreamer/nnstreamer_cppplugin_api_filter.hh
+/usr/include/nnstreamer/ml-api-common.h
+/usr/include/nnstreamer/tizen_error.h
+/usr/include/nnstreamer/nnstreamer.h
+/usr/include/nnstreamer/nnstreamer-single.h
 /usr/lib/*/pkgconfig/*.pc
 /usr/lib/*/*.a

--- a/debian/nnstreamer.install
+++ b/debian/nnstreamer.install
@@ -5,5 +5,5 @@
 /usr/lib/nnstreamer/decoders/libnnstreamer_decoder_direct_video.so
 /usr/lib/nnstreamer/filters/libnnstreamer_filter_cpp.so
 /usr/lib/*/gstreamer-1.0/*.so
-/usr/lib/*/libcapi-*.so
+/usr/lib/*/libcapi-*
 /etc/nnstreamer.ini

--- a/debian/rules
+++ b/debian/rules
@@ -37,6 +37,7 @@ endif
 
 override_dh_auto_clean:
 	rm -rf ${BUILDDIR}
+	rm -rf tools/development/confchk/${BUILDDIR}
 
 override_dh_auto_configure:
 	mkdir -p ${BUILDDIR}
@@ -44,7 +45,7 @@ override_dh_auto_configure:
 	-Dtf-support=$(enable_tf) -Dtflite-support=enabled -Dpytorch-support=enabled -Dcaffe2-support=enabled \
 	-Dpython2-support=enabled -Dpython3-support=enabled -Denable-capi=true -Denable-edgetpu=true -Denable-tizen=false \
 	-Denable-openvino=true ${BUILDDIR}
-	cd tools/development/confchk && meson --prefix=/usr ${BUILDDIR} && cd ../../..
+	cd tools/development/confchk && meson --prefix=/usr --bindir=bin ${BUILDDIR} && cd ../../..
 
 override_dh_auto_build:
 	ninja -C ${BUILDDIR}

